suite-common/wallet-core/src/transactions/transactionsReducer.ts

```diff
@@ -6,9 +6,9 @@ import {
     findTransaction,
     getConfirmations,
     isPending,
-    getIsPhishingTransaction,
     getEverstakePool,
 } from '@suite-common/wallet-utils';
+import { getIsPhishingTransaction } from '@suite-common/token-definitions/src/antiFraud';
```

Comment:
export it in root so that you can import it from `@suite-common/token-definitions`